components/blitz/src/omero/gateway/SecurityContext.java

```diff
@@ -43,8 +43,10 @@
     /** Indicates to generate session for another user. */
     private boolean sudo;
 
+    /** Host/port of the server */
     private ServerInformation serverInformation;
 
+    /** The compression level */
```

Comment:
Can we say what the valid range is and what it means? (If not, that's okay, don't spend too much time researching!)